locallib.php

```diff
@@ -267,10 +267,10 @@ function bigbluebuttonbn_get_recordings_array_fetch($meetingidsarray) {
  *
  * @return array
  */
-function bigbluebuttonbn_get_recordings_array_fetch_page($mids) {
+function bigbluebuttonbn_get_recordings_array_fetch_page($mids, $rids = []) {
     $recordings = array();
     // Do getRecordings is executed using a method GET (supported by all versions of BBB).
-    $url = \mod_bigbluebuttonbn\locallib\bigbluebutton::action_url('getRecordings', ['meetingID' => implode(',', $mids)]);
+    $url = \mod_bigbluebuttonbn\locallib\bigbluebutton::action_url('getRecordings', ['meetingID' => implode(',', $mids), 'recordID' => implode(',', $rids)]);
```

Comment:
Just a thought, should the recordID param still be provided if the array parameter is empty? likely harmless

Reply:
tested passing it empty has no adverse effects